zip_maker.rb

```diff
@@ -224,9 +224,8 @@ def add_dir(src, dst, recursive, zipfile)
             if !exclude
               dst_file = src_file
               if element_present?(file_entry, 'dst_path')
-                split_file = dst_file.split('/')
-                dst_file = split_file[1..split_file.length].join('/')
-                dst_file = File.join(file_entry['dst_path'], dst_file)
+                split_file = dst_file.split('/')[-1]
```

Comment:
Could probably just use `File.basename(dst_file)` instead of splitting, but this works.